### PR TITLE
InMobi Ad extension for AMP

### PR DESCRIPTION
**Pull request by nkapser**
Hi Team,

Please find the InMobi Ad Extension for AMP.

Please review the same and add comments. My manager is filling the required CLA.

Thanks

### CODE REVIEW

**Comment by lannka** [comment]
we recently added new API: [renderStart](https://github.com/ampproject/amphtml/blob/master/ads/README.md#methods-available-to-the-ad)
Could you take a look and implement the APIs (renderStart + noContentAvailable)?
It helps AMP to provide a better ad loading experience.
Let me know if you have any questions regarding that.

**Comment by nkapser** [comment]
@lannka Sure. I will take a look it and implement it.

**Comment by lannka** [comment]
@nkapser just a heads up: you might want to sync to head to resolve some conflicts before implementing the new APIs.

**Comment by nkapser** [comment]
@lannka I have sync'ed with the latest codebase. Included renderStart and noContentAvailable for NFR.

**Comment by nkapser** [comment]
I signed it!

File: 3p/integration.js [hunk 1]
```diff
@@ -208,6 +208,7 @@ const defaultAllowedTypesInCustomFrame = [
 export const waitForRenderStart = [
   'doubleclick',
   'fakead3p',
+  'inmobi',
```
Comment (lannka) [inline 3p/integration.js]: did you sync to the latest code? we recently moved this array to a different file.

Comment (lannka) [inline 3p/integration.js]: Never mind, I was looking at a wrong diff

**Comment by nkapser** [comment]
Hi @lannka, My manager has singed the CLA and I am part of the group. 
Where do we see the error for cla check ?

File: ads/inmobi.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (lannka) [inline ads/inmobi.md]: 2016

File: ads/inmobi.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+# InMobi
+
+## Examples
+
+### For 320x50 Ad
+
+```html
+<amp-ad
+width="320"
```
Comment (lannka) [inline ads/inmobi.md]: please add 4-space indentation

File: builtins/amp-ad.md [hunk 4]
```diff
@@ -103,6 +103,7 @@ resources in AMP. It requires a `type` argument that select what ad network is d
 - [I-Mobile](../ads/imobile.md)
 - [Improve Digital](../ads/improvedigital.md)
 - [Industrybrains](../ads/industrybrains.md)
+- [InMobi](../ads/inmobi.md)
```
Comment (lannka) [inline builtins/amp-ad.md]: this file is deprecated as will soon be removed.
would you add to `extensions/amp-ad/amp-ad.md`

File: examples/ads.amp.html [hunk 5]
```diff
@@ -539,6 +540,17 @@ <h2 id="industrybrains">Industrybrains</h2>
     <div fallback></div>
   </amp-ad>
 
+  <h2 id="inmobi">InMobi</h2>
+  <amp-ad
+    width="320"
```
Comment (lannka) [inline examples/ads.amp.html]: nitpick, to keep a consistency, could you move `width` and `height` to the previous line?

File: builtins/amp-ad.md (hunk 4 above)
Comment (nkapser) [inline builtins/amp-ad.md]: Added Inmobi to extensions/amp-ad/amp-ad.md

Comment (nkapser) [inline builtins/amp-ad.md]: done.

File: examples/ads.amp.html (hunk 5 above)
Comment (nkapser) [inline examples/ads.amp.html]: done.

File: ads/inmobi.md (hunk 3 above)
Comment (nkapser) [inline ads/inmobi.md]: done.

File: ads/inmobi.md (hunk 2 above)
Comment (nkapser) [inline ads/inmobi.md]: done.

**Pull request merged**